packages/create-app/templates/default-app/packages/app/package.json.hbs

```diff
@@ -15,7 +15,6 @@
     "@backstage/plugin-scaffolder": "^{{version '@backstage/plugin-scaffolder'}}",
     "@backstage/plugin-techdocs": "^{{version '@backstage/plugin-techdocs'}}",
     "@backstage/catalog-model": "^{{version '@backstage/catalog-model'}}",
-    "@backstage/plugin-circleci": "^{{version '@backstage/plugin-circleci'}}",
```

Comment:
iirc there's a peerDependency that should be removed from from create-app as well

Reply:
Thanks, removed it!